packages/manager/modules/ng-layout-helpers/src/list/list-layout.controller.js

```diff
@@ -39,8 +38,19 @@ export default class ListLayoutCtrl {
     return this.sort === property ? this.sortOrder.toLowerCase() : '';
   }
 
+  getDisplayedColumns(columns) {
+    this.displayedColumns = columns
+      ? JSON.stringify(
+          map(
```

Comment:
Native map is used before, is this usefull to import Lodash one for this one ? perf issue ?

Reply:
Lodash handles more cases like if the parameter is undefined and stuff like that